umbraco-cloud/set-up/legacy-umbraco-visual-studio-setup.md

```diff
@@ -0,0 +1,212 @@
+
+If you haven't cloned the repository before or don't have a [git credentials manager](https://github.com/Microsoft/Git-Credential-Manager-for-Windows) installed you will be asked to enter the username and password for the Umbraco Cloud Project. These are the same credentials as you use to access the Portal and the Umbraco backoffice.
+
+![cmd clone](images/cmd-clone.png)
+
+Once it's done running the tool will have created a Visual Studio solution file `*.sln` and two Projects.
+
+* `*.Web` contains the Umbraco site that was (git) cloned from your Project
+* `*.Core` is a Class Library that you can use for your custom code, as mentioned above
+
+Both projects are configured with the NuGet packages for Umbraco using the version that corresponds to the site cloned from Umbraco Cloud.
+
+The result should look something like this within the folder where the UaaS.cmd tool ran:
+
+![Generated Solution](images/generated-solution.png)
+
+You can now open the solution in Visual Studio and hit F5 to start the site directly from Visual Studio.
+
+## The Git repositories
+One thing to notice about this setup is that you will get two git repositories as well as two projects.
+
+1. The site cloned from your Umbraco Cloud Project will be contained within a git repository that is connected to your Project on Umbraco Cloud. Whenever you want to deploy changes to your (remote) Umbraco Cloud site you should commit everything within the `*.Web` folder, which is where the git repository for Umbraco Cloud is also located.
+
+2. Going up one level to where the `*.sln` file is located you will notice a `.git` folder. This is the second git repository. You should use this repository for all the code you write as well as the solution and project files for Visual Studio.
+
+Think of everything within the `*.Web` folder as your **deployment repository**, and everything surrounding that folder as your **source code repository**. The Umbraco Cloud repository (within the `*.Web` folder) will not (and should not) be committed to the other git repository.
+
+## What's next?
+
+Now that you've added your own touch to your site, you're ready to deploy to your Umbraco Cloud environment.
+The key thing to know is that your custom code from the `*.Core` project will be built into a .dll file in your `*.Web` project that you can then push up to the Cloud repository.
+
+Once you have everything your site will need committed you can follow the [deployment workflow](/umbraco-cloud/deployment/README.md) to complete the deployment.
+
+## Working with Visual Studio
+As mentioned in the previous section, you will start with two projects in Visual Studio. A project called `*.Web` with the Umbraco site (from Umbraco Cloud) configured as a Website project, and a project called `*.Core` configured as a class library for all of your code.
```

Comment:
This might remove too much of the meaning - what do you think?

```suggestion
As mentioned in the previous section, you will start with two projects in Visual Studio.  A `*.Web` project with the Umbraco site configured as a Website project, and a `*.Core` project configured as a class library.
```